tsl/src/bgw_policy/reorder_api.c

```diff
@@ -202,20 +216,6 @@ policy_reorder_add(PG_FUNCTION_ARGS)
 	namestrcpy(&proc_schema, INTERNAL_SCHEMA_NAME);
 	namestrcpy(&owner, GetUserNameFromId(owner_id, false));
 
-	/*
-	 * Try to see if the hypertable has a specified chunk length for the
-	 * default schedule interval
-	 */
-	dim = hyperspace_get_open_dimension(ht->space, 0);
-
-	partitioning_type = ts_dimension_get_partition_type(dim);
-	if (dim && IS_TIMESTAMP_TYPE(partitioning_type))
-	{
-		schedule_interval.time = dim->fd.interval_length / 2;
-		schedule_interval.day = 0;
-		schedule_interval.month = 0;
-	}
-
 	JsonbParseState *parse_state = NULL;
```

Comment:
The Hypertable object (`ht` pointer) is still used a few lines below after releasing the cache.

Reply:
good catch